src/redisenterprise/azext_redisenterprise/generated/custom.py

```diff
@@ -61,17 +66,22 @@ def redisenterprise_create(client,
 def redisenterprise_update(client,
                            resource_group_name,
                            cluster_name,
-                           sku=None,
+                           sku,
```

Comment:
Making `sku` required parameter is also breaking change, can you double confirm?

Reply:
It should be optional for incremental updates to existing caches, using PATCH verb.

Reply:
Changed `sku=None` to `sku` will make `--sku` parameter required for `az redisenterprise update`. Can you check with `az redisenterprise update -h`?

Reply:
Since sku was incorrectly generated as mandatory from our swagger, made sku as optional in manual/custom.py